frontend/pages/ManageControlsPage/OSUpdates/components/EndUserOSRequirementPreview/EndUserOSRequirementPreview.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+
+import CustomLink from "components/CustomLink";
+
+import { OSUpdatesSupportedPlatform } from "../../OSUpdates";
+
+import MacOSUpdateScreenshot from "../../../../../../assets/images/macos-updates-preview.png";
+import WindowsUpdateScreenshot from "../../../../../../assets/images/windows-nudge-screenshot.png";
+
+const baseClass = "os-requirement-preview";
+
+interface IOSRequirementDescriptionProps {
+  platform: OSUpdatesSupportedPlatform;
+}
+const OSRequirementDescription = ({
```

Comment:
I think we could stand to save a few lines here by encapsulating common markdown, wdyt?

Reply:
Still something we can keep in mind going forward